src/demo.cpp

```diff
@@ -1,17 +1,18 @@
 /** This is a very ugly test code (doomed to fail linting) */
 #include "demo.hpp"
-#include <cstdio>
-#include <cstddef>
+#include <stdio.h>
 
```

Comment:
patch for this change is
```diff
@@ -3 +3 @@
-#include <stdio.h>
+#include <cstdio>
```
but the comment here is for line 3-4:
![image](https://github.com/cpp-linter/test-cpp-linter-action/assets/14963867/2e13b4be-6c12-4d90-becc-22f32daafa82)

Again, I think the hunk header (`@@ -3 +3 @@`) is causing a problem since it doesn't have the number of lines in the hunk.